README.md

```diff
@@ -67,6 +68,36 @@
 - [NEPs](https://github.com/neo-project/proposals) - NEPs describe standards for the NEO platform.
 - [Network Protocol](http://docs.neo.org/en-us/node/network-protocol.html) - Explanation of how NEO nodes communicate.
 
+## Exchanges
+
+- [Anycoindirect](https://anycoindirect.eu/en/buy/neo) - A Dutch exchange that accepts iDeal payments for **NEO**.  
+*English, Dutch, German, French, Spanish and Italian interfaces*
+- [Binance](https://www.binance.com/tradeDetail.html?symbol=NEO_BTC) - A Hong Kong-based exchange that allows trading **NEO** and **GAS** with BTC. Will return GAS when you hold NEO on their exchange.  
+*English, Chinese, Korean, Japanese, Russian and Spanish interfaces*
+- [Bitfinex](https://www.bitfinex.com/stats) - A Hong Kong-based exchange where you can trade **NEO** with BTC.  
```

Comment:
Please see CONTRIBUTING.md about not including `NEO` in the description of links.